package.json

```diff
@@ -26,7 +26,7 @@
     "grunt-ng-annotate": "2.0.2",
     "grunt-ngdocs": "0.2.10",
     "grunt-svgmin": "3.3.0",
-    "grunt-usemin": "3.1.1",
```

Comment:
This changes from an exact version to a floating version which is a regression, so let's revert this.